java-grupa8/homework-1/tema/src/ro/siit/tema/BubbleSort.java

```diff
@@ -0,0 +1,29 @@
+package ro.siit.tema;
+
+/**
+ * Created by x-RauL-x on 7/26/2017.
+ */
+public class BubbleSort {
+    /**
+     * A method used to bubble sort an array of objects
+     * @param SalesGuy object of the class SalesRepresentatives
+     */
+    public static void bubbleSort(SalesRepresentatives[] SalesGuy) {
+        int swap;
```

Comment:
You could have used a boolean value (i.e isSorted).
Initially you set it to true. Once a swap happens you set it to false.